packages/react/src/number-field/scrub-area-cursor/NumberFieldScrubAreaCursor.tsx

```diff
@@ -31,7 +31,10 @@ export const NumberFieldScrubAreaCursor = React.forwardRef(function NumberFieldS
 
   const [element, setElement] = React.useState<Element | null>(null);
 
-  const renderElement = useRenderElementLazy('span', componentProps, {
+  const shouldRender = isScrubbing && !isWebKit && !isTouchInput && !isPointerLockDenied;
+
+  const renderedElement = useRenderElement('span', componentProps, {
```

Comment:
The rest of the codebase uses `element` consistently as the name for the variable, as far as I remember.